src/cmd/src/standalone.rs

```diff
@@ -105,7 +106,7 @@ impl StandaloneOptions {
             prometheus_options: self.prometheus_options,
             mode: self.mode,
             datanode_rpc_addr: "127.0.0.1:3001".to_string(),
-            metasrv_addr: None,
+            meta_client_opts: Some(MetaClientOpts::default()),
```

Comment:
Is meta required in standalone mode?

Reply:
> Is meta required in standalone mode?

No

Reply:
>No

Then how about default this field to `None`